inference-engine/src/vpu/graph_transformer/src/stages/static_shape_nms.cpp

```diff
@@ -53,29 +84,45 @@ class StaticShapeNMS final : public NonMaxSuppression {
 }  // namespace
 
 void FrontEnd::parseStaticShapeNMS(const Model& model, const ie::CNNLayerPtr& layer, const DataVector& inputs, const DataVector& outputs) const {
-    VPU_THROW_UNLESS(inputs.size() >= 2 && inputs.size() <= 5,
-        "StaticShapeNMS parsing failed, expected number of input is in range [2, 5], but {} provided",
-        inputs.size());
-    VPU_THROW_UNLESS(outputs.size() == 2,
-        "StaticShapeNMS parsing failed, expected number of outputs: 2, but {} provided",
-        outputs.size());
+    VPU_THROW_UNLESS(inputs.size() == 6,
```

Comment:
I'm not sure all 6 inputs are required. In specification only first two are marked as `Required` and an example in doc has 5 inputs

Reply:
But we will always get 6 inputs, because the NMS-5 always creates default values for inputs that are not presented in the network.

Reply:
Ok, thanks for explanation!